src/server/enrollment-system.js

```diff
@@ -88,6 +88,37 @@ function spanishHispanicToSDSCode(isSpanishHispanicLatino) {
 }
 
 /**
+ * Returns SDS code representing the indication of Spanish/Hispanic/Latino ethnicity.
```

Comment:
Haha copy paste of a method above to save writing so little. Will update then merge